backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -306,3 +310,28 @@ fieldset {
 .form-buttons {
   text-align: center;
 }
+
+.form-section {
+  @include make-row(0px);
+  margin: 10px 0px;
+  padding: 10px 0px;
+  background: very-light($color-3, 4);
+  border: 1px solid very-light($color-3, 32);
+}
+
+.text-field-addon {
+  position: absolute;
+  right: 25px;
+  bottom: 21px;
+  pointer-events: none;
+}
+
+.max-width-form {
+  @include margin(null null auto null);
+  max-width: map-get($container-max-widths, "xl");
+}
+
+.helpful-text-sidebar {
+  margin: 10px 0px;
+  padding: 10px 0px;
+}
```

Comment:
I think that this one can also be classified as a `form-section` or `card`, just without background and border. So, maybe something like:

```css
.card.light {
  border: 0;
}
```

If we minimize those "public api" classes, it will be easier for other developers to make use of them.